homeassistant/components/system_bridge/__init__.py

```diff
@@ -0,0 +1,261 @@
+"""The System Bridge integration."""
+import asyncio
+from datetime import timedelta
+import logging
+import shlex
+from typing import Any, Dict, Optional
+
+import async_timeout
+from systembridge import Bridge
+from systembridge.client import BridgeClient
+from systembridge.exceptions import BridgeAuthenticationException
+from systembridge.objects.command.response import CommandResponse
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_COMMAND,
+    CONF_HOST,
+    CONF_PATH,
+    CONF_PORT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady
+from homeassistant.helpers import (
+    aiohttp_client,
+    config_validation as cv,
+    device_registry as dr,
+)
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import BRIDGE_CONNECTION_ERRORS, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
+
+CONF_ARGUMENTS = "arguments"
+CONF_BRIDGE = "bridge"
+CONF_WAIT = "wait"
+
+SERVICE_SEND_COMMAND = "send_command"
+SERVICE_SEND_COMMAND_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_BRIDGE): cv.string,
+        vol.Required(CONF_COMMAND): cv.string,
+        vol.Optional(CONF_ARGUMENTS, []): cv.string,
+    }
+)
+SERVICE_OPEN = "open"
+SERVICE_OPEN_SCHEMA = vol.Schema(
+    {vol.Required(CONF_BRIDGE): cv.string, vol.Required(CONF_PATH): cv.string}
+)
+
+
+async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
+    """Set up the System Bridge integration."""
+
+    async def handle_send_command(call):
+        """Handle the send_command service call."""
+        device_registry = await hass.helpers.device_registry.async_get_registry()
```

Comment:
Please use `async_get` in the device registry helper module to get the registry. That is not a coroutine function

Reply:
Bump.